src/controllers/words.js

```diff
@@ -64,18 +64,39 @@ export const getWords = async (req, res, next) => {
       dialects,
       examples,
     };
-    let query = !strict ? searchIgboTextSearch(searchWord, isUsingMainKey) : strictSearchIgboQuery(searchWord);
-    const words = await searchWordUsingIgbo({ query, ...searchQueries });
-    if (!words.length) {
-      query = searchEnglishRegexQuery(regexKeyword);
-      const englishWords = await searchWordUsingEnglish({ query, ...searchQueries });
-      return packageResponse({
-        res,
-        docs: englishWords,
-        model: Word,
-        query,
-        ...rest,
-      });
+    let words;
+    let query;
+    if (searchWord.match(/".*"/) || searchWord.match(/'.*'/)) {
```

Comment:
So it looks like what you want to do instead, is to remove the quotes from `searchWord` before passing it into `handleQueries` so that the returned `regexKeyword` doesn't include quotes. This will allow you to omit the lines 70 to 76 that you've written since you don't need to convert the regex into a string and back into a regex.

So these are the steps you want to take:
1. Before calling `handleQueries` on line 59, check to see if `searchWord` has either single or double quotes. If it does, keep track of that state by storing the boolean value in a variable (maybe called `hasQuotes`), we will need to know about this later.
2. Remove the quotes from `searchWord`, specifially from `req.searchWord`,  and then call `handleQueries` by passing in `req` that has the newly updated `searchWord` that doesn't have any quotes.
3. This will return `regexKeyword` which is a regular expression that doesn't include any quotes, so we don't have to remove those quotes from the regex later.
4. After we get `regexKeyword` we want to know whether or not our original `searchWord` had quotes or not. If it did, then we want to call `query = searchEnglishRegexQuery()` and return whatever result we get. If it didn't have quotes, then we want search with Igbo first.